database/dynamodb.go

```diff
@@ -38,44 +38,42 @@ func getAWSCredentials() *AWSCredentials {
 	if !found {
 		log.Println("AWS region not stored, please store it.")
 
-		// Here the error is passed for debugging to the user we show "Something went wrong"
-		utils.ServerError(errors.New("AWS region not stored in ENV var")) 
+		utils.ServerError(errors.New("AWS region not stored in ENV var"))
 	}
 
 	awsCredentials.AccessKey, found = os.LookupEnv(utils.ACCESS_KEY)
 	if !found {
 		log.Println("AWS Access key not stored, please store it.")
-		
-		// Here the error is passed for debugging to the user we show "Something went wrong"
+
 		utils.ServerError(errors.New("AWS Access key not stored in ENV var"))
 	}
 
 	awsCredentials.SecretKey, found = os.LookupEnv(utils.SECRET_KEY)
 	if !found {
 		log.Println("AWS Secret key not stored, please store it.")
 
-		// Here the error is passed for debugging to the user we show "Something went wrong
 		utils.ServerError(errors.New("AWS Secret key not stored, please store it."))
 	}
 
 	return awsCredentials
 }
 
-func GetFeatureFlagTableName() string {
-	tableName, found := os.LookupEnv(utils.FF_TABLE_NAME)
+func GetTableName(envVarName string) string {
+	tableName, found := os.LookupEnv(envVarName)
 	if !found {
-		log.Println("Feature Flag table name is not set in env.")
+		errorMessage := fmt.Sprintf("%v is not set in env. \n", envVarName)
+
+		log.Printf(errorMessage)
 
-		// Here the error is passed for debugging to the user we show "Something went wrong
-		utils.ServerError(errors.New("Feature flag table name not stored in the env."))
+		utils.ServerError(errors.New(errorMessage))
 	}
 	return tableName
 }
 
 func CreateDynamoDB() *dynamodb.DynamoDB {
 	defer func() {
 		if err := recover(); err != nil {
-			log.Printf("Error is %v",err)
+			log.Printf("Error is %v", err)
```

Comment:
i would suggest show the error in next line, if the error is multi line the formatting will go off

Reply:
That's a good point. Updated the code